editor/loadFile.js

```diff
@@ -0,0 +1,26 @@
+function createFileLoader (moonchild, channel) {
+  var fileLoader = Object.create(null);
+
+  // assuming that the editor doesn't get changed
+  // weird stuff will happen if the editor -does- get changed
+  var editor = moonchild.getEditor()._codeMirror;
+  var filePath;
+
+  channel.on("fileLoad", function (messageData) {
+    // assuming JSON structure of form
+    // {
+    //   ...
+    //   content: "",
+    //   filePath: "bla.js"
+    //   ...
+    // }
+
+    var text = messageData.content;
+    filePath = messageData.filePath;
+
+    editor.setValue(text);
+
+    // let the editor know that a change happened
+    moonchild.poke();
```

Comment:
Ah nevermind, after cleaning it up a little bit, there's only 2 lines of code left, one of which is redundant when moved to editor.js. In this case I think it's fine to move it to editor.js